subprojects/packagefiles/tinycdb/meson.build

```diff
@@ -0,0 +1,41 @@
+# SPDX-License-Identifier: LGPL-2.1-or-later
```

Comment:
This means the `tinycdb` directory needs a `LICENSE.build` file, or else the patch ZIP will get a default one with the MIT license.

Reply:
Added